src/styles/fonts.css

```diff
@@ -33,12 +33,12 @@
   font-weight: 900;
   font-family: "Lato";
   font-style: normal;
-  font-display: swap;
+  font-display: fallback;
   src: url("../fonts/lato-v20-latin-900.eot"); /* IE9 Compat Modes */
   src: local(""),
     url("../fonts/lato-v20-latin-900.eot?#iefix") format("embedded-opentype"),
-    url("../fonts/lato-v20-latin-900.woff2") format("woff2"),
```

Comment:
please, make sure to remove these from the repo

Reply:
I left here, if needed in future, so we can regenerate the new font subset